sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientTestBase.java

```diff
@@ -0,0 +1,1014 @@
+        TextDocumentStatistics textDocumentStatistics2 = new TextDocumentStatistics().setCharacterCount(21).setTransactionCount(1);
+
+        KeyPhraseResult keyPhraseResult1 = new KeyPhraseResult("0", textDocumentStatistics1, null, keyPhrasesList1);
+        KeyPhraseResult keyPhraseResult2 = new KeyPhraseResult("1", textDocumentStatistics2, null, keyPhrasesList2);
+
+        TextBatchStatistics textBatchStatistics = new TextBatchStatistics().setDocumentCount(2).setErroneousDocumentCount(0).setTransactionCount(2).setValidDocumentCount(2);
+        List<KeyPhraseResult> keyPhraseResultList = Arrays.asList(keyPhraseResult1, keyPhraseResult2);
+
+        return new DocumentResultCollection<>(keyPhraseResultList, MODEL_VERSION, textBatchStatistics);
+    }
+
+    static DocumentResultCollection<TextSentimentResult> getExpectedBatchTextSentiment() {
+        final TextDocumentStatistics textDocumentStatistics1 = new TextDocumentStatistics().setCharacterCount(67)
+            .setTransactionCount(1);
+        final TextDocumentStatistics textDocumentStatistics2 = new TextDocumentStatistics().setCharacterCount(67)
+            .setTransactionCount(1);
+
+        final TextSentiment expectedDocumentSentiment = new TextSentiment()
+            .setTextSentimentClass(TextSentimentClass.MIXED).setLength(66).setOffset(ZERO)
+            .setNegativeScore(0.00019).setNeutralScore(0.5).setPositiveScore(0.4);
+
+        final TextSentimentResult textSentimentResult1 = new TextSentimentResult("0", textDocumentStatistics1,
+            null,
+            expectedDocumentSentiment,
+            Arrays.asList(
+                new TextSentiment().setTextSentimentClass(TextSentimentClass.NEGATIVE).setLength(31).setOffset(0)
+                    .setNegativeScore(0.99),
+                new TextSentiment().setTextSentimentClass(TextSentimentClass.POSITIVE).setLength(35).setOffset(32)
+                    .setPositiveScore(0.99)
+            ));
+
+        final TextSentimentResult textSentimentResult2 = new TextSentimentResult("1", textDocumentStatistics2,
+            null,
+            expectedDocumentSentiment,
+            Arrays.asList(
+                new TextSentiment().setTextSentimentClass(TextSentimentClass.POSITIVE).setLength(35).setOffset(0)
+                    .setPositiveScore(0.99),
+                new TextSentiment().setTextSentimentClass(TextSentimentClass.NEGATIVE).setLength(31).setOffset(36)
+                    .setNegativeScore(0.99)
+            ));
+
+        return new DocumentResultCollection<>(Arrays.asList(textSentimentResult1, textSentimentResult2),
+            MODEL_VERSION,
+            new TextBatchStatistics().setDocumentCount(2).setErroneousDocumentCount(0)
+                .setTransactionCount(2).setValidDocumentCount(2));
+    }
+
+    private TextAnalyticsRequestOptions setTextAnalyticsRequestOptions() {
+        this.showStatistics = true;
```

Comment:
Why a member variable to control it? What happens if we run tests in parallel? i think it should be another parameter on validation.